cmd/xkcd-gtk/style.go

```diff
@@ -157,11 +158,11 @@ func (win *Window) StyleUpdated() {
 		}
 	}
 
-	menuPopoverChild, err := win.menu.GetPopover().GetChild()
+	c, err := win.menu.GetPopover().GetChild()
 	if err != nil {
 		log.Print(err)
 	} else {
-		menuBox := (&gtk.Stack{Container: gtk.Container{Widget: *menuPopoverChild}}).GetVisibleChild()
+		menuBox := c.ToWidget()
```

Comment:
I'm not entirely sure about this change. Seems to work as expected, but didn't understand the motivation behind the original code.

Reply:
This code isn't great and I ended up removing it in head due to version difficulties maintaining it. I might reintroduce it later before the next release, but for now it is gone.